src/main/java/org/verdictdb/connection/ConcurrentJdbcConnection.java

```diff
@@ -133,6 +133,11 @@ public void setDefaultSchema(String schema) throws VerdictDBDbmsException {
     }
   }
 
+  @Override
+  public List<String> getPrimaryKey(String schema, String table) throws VerdictDBDbmsException {
+    return null;
```

Comment:
Please just call
`return getNextConnection().getPrimaryKey(schema, table);`

Reply:
I have changed this line.